backend/src/controllers/users/UsersController.js

```diff
@@ -67,14 +67,20 @@ module.exports = {
     try {
       const allowedUser = await bcrypt.compare(user.password, foundUser.password);
       if (!allowedUser) throw new Error('wrong credentials');
-      const token = await createJwt({ role: foundUser.role, email: foundUser.email, promoId: foundUser.promotionId });
-      return res
-        .cookie('token', token, {
-          maxAge: 900000,
-          httpOnly: true
-        })
-        .status(200)
-        .send({token});
+      createJwt({ role: foundUser.role, email: foundUser.email, promoId: foundUser.promotionId }, (err, token) => {
```

Comment:
Looks like the payload should be extracted into a var, since it is reused below for the response body.